{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -88,10 +91,12 @@ sync-from-source: ## download data data source to local envrionment
 {% endif %}
 
 init-docker: ## initialize docker image
+	$(DOCKER) build -t $(BASE_IMAGE_NAME) -f $(BASE_DOCKERFILE) --build-arg UID=$(shell id -u) .
 	$(DOCKER) build -t $(IMAGE_NAME) -f $(DOCKERFILE) --build-arg UID=$(shell id -u) .
```

Comment:
Do you need this `--build-arg`? 
It occurs this warning `[Warning] One or more build-args [UID] were not consumed`.
It does not seem to matter if you clear it.

Reply:
👍 fixed.